src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -3056,22 +3059,24 @@ static void window_top_toolbar_tool_abort(rct_window *w, rct_widgetindex widgetI
 }
 
 static void top_toolbar_init_fastforward_menu(rct_window* w, rct_widget* widget) {
-    int32_t num_items = 4;
+    int32_t num_items = 5;
     gDropdownItemsFormat[0] = STR_TOGGLE_OPTION;
     gDropdownItemsFormat[1] = STR_TOGGLE_OPTION;
     gDropdownItemsFormat[2] = STR_TOGGLE_OPTION;
     gDropdownItemsFormat[3] = STR_TOGGLE_OPTION;
+    gDropdownItemsFormat[4] = STR_TOGGLE_OPTION;
     if (gConfigGeneral.debugging_tools) {
-        gDropdownItemsFormat[4] = STR_EMPTY;
-        gDropdownItemsFormat[5] = STR_TOGGLE_OPTION;
-        gDropdownItemsArgs[5] = STR_SPEED_HYPER;
-        num_items = 6;
+        gDropdownItemsFormat[5] = STR_EMPTY;
+        gDropdownItemsFormat[6] = STR_TOGGLE_OPTION;
+        gDropdownItemsArgs[6] = STR_SPEED_HYPER;
+        num_items = 7;
     }
 
-    gDropdownItemsArgs[0] = STR_SPEED_NORMAL;
-    gDropdownItemsArgs[1] = STR_SPEED_QUICK;
-    gDropdownItemsArgs[2] = STR_SPEED_FAST;
-    gDropdownItemsArgs[3] = STR_SPEED_TURBO;
+    gDropdownItemsArgs[0] = STR_SPEED_SLOW;
+    gDropdownItemsArgs[1] = STR_SPEED_NORMAL;
+    gDropdownItemsArgs[2] = STR_SPEED_QUICK;
+    gDropdownItemsArgs[3] = STR_SPEED_FAST;
+    gDropdownItemsArgs[4] = STR_SPEED_TURBO;
```

Comment:
This entire block can be improved by taking the values from a static array instead, to get rid of the hardcoded numbers and string IDs.
Edit: The array already exists: `SpeedNames`